### PR TITLE
add validation on update status as created to return also as updating…

### PR DESCRIPTION
**Pull request by acosferreira**
…, given process started

# Description
add status validation to updating

FIXES: 

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

### CODE REVIEW

File: pkg/services/devices.go [hunk 1]
```diff
@@ -826,7 +826,7 @@ func ReturnDevicesView(storedDevices []models.Device, orgID string) ([]models.De
 				}
 			}
 
-			if updateStatus == models.UpdateStatusBuilding {
+			if updateStatus == models.UpdateStatusBuilding || updateStatus == models.UpdateStatusCreated {
```
Comment (mgold1234) [inline pkg/services/devices.go:829]: is it possible to check this condition of update status created?

Comment (ldjebran) [reply pkg/services/devices.go:829]: think we should add a unittest for this (this is changing our contract to UI).

Comment (acosferreira) [reply pkg/services/devices.go:829]: Added the test scenarios to cover this change

**Review by ldjebran — APPROVED** [review]
lgtm

**Comment by adeliaferreira** [comment]
lgtm\

**Review by adeliaferreira — APPROVED** [review]
lgtm

**Pull request merged**